src/pm-conversations/__tests__/pmConversationsSelectors-test.js

```diff
@@ -23,103 +27,117 @@ describe('getRecentConversations', () => {
   });
 
   test('returns unique list of recipients, includes conversations with self', () => {
-    const state = deepFreeze({
-      realm: { email: 'me@example.com' },
-      users: [
-        { user_id: 0, email: 'me@example.com' },
-        { user_id: 1, email: 'john@example.com' },
-        { user_id: 2, email: 'mark@example.com' },
+    const meAndJohnPm1 = eg.pmMessage({
+      id: 1,
+      display_recipient: [
+        eg.displayRecipientFromUser(eg.selfUser),
+        eg.displayRecipientFromUser(userJohn),
+      ],
+    });
+
+    const meAndMarkPm = eg.pmMessage({
+      id: 2,
+      display_recipient: [
+        eg.displayRecipientFromUser(eg.selfUser),
+        eg.displayRecipientFromUser(userMark),
       ],
+    });
+
+    const meAndJohnPm2 = eg.pmMessage({
+      id: 3,
+      display_recipient: [
+        eg.displayRecipientFromUser(eg.selfUser),
+        eg.displayRecipientFromUser(userJohn),
+      ],
+    });
+
+    const meOnlyPm = eg.pmMessage({
+      id: 4,
+      display_recipient: [eg.displayRecipientFromUser(eg.selfUser)],
+    });
+
+    const meJohnAndMarkPm = eg.pmMessage({
+      id: 0,
+      display_recipient: [
+        eg.displayRecipientFromUser(eg.selfUser),
+        eg.displayRecipientFromUser(userMark),
+        eg.displayRecipientFromUser(userJohn),
+      ],
```

Comment:
These are probably a sign that it'd be helpful to refactor `eg.displayRecipientFromUser` to be called, say, `eg.displayRecipientFromUsers` -- and to take `User[]` and return `PmRecipientUser[]`, and be just a `map` around what it currently is.

(Maybe even both versions should exist.)